7.1 дз/7.1 дз/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Homework
+{
+
+    class Program
+    {
+
+        public bool CountNum(double m, double n)
+        {
+            if (m > 0 | n > 0)
```

Comment:
Лучше использовать ленивую дизъюнкцию
```suggestion
            if (m > 0 || n > 0)
```